Sources/ArcGISToolkit/Components/Augmented Reality/TableTopSceneView.swift

```diff
@@ -17,7 +17,6 @@ import SwiftUI
 import ArcGIS
 
 /// A scene view that provides an augmented reality table top experience.
-@MainActor
 @preconcurrency
```

Comment:
Please remove this attribute as well.

Reply:
This is confusing, this attribute [is not there](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/27ce6fc7e2c5fcd61a62236e1590df1f7bba074e/Sources/ArcGISToolkit/Components/Augmented%20Reality/TableTopSceneView.swift#L20-L21) and the feedback comment doesn't have a yellow "Outdated" tag. 😆